mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -440,4 +441,11 @@ public Flags getFlags() {
         MailboxACL getACL();
 
     }
+
+    /**
+     * Get resolved ACL on this Mailbox for the given Session
+     *
+     * The result will be the same as calling {MessageManager#getMetaDtata().getAcl()} but will load fewer data
```

Comment:
Oh, really, how could this be ?

Reply:
because getMetaData retrieves more metadata than just getAcl?

Reply:
It makes sense